src/meetings/components/meetingAccessPoint/mediaHandlers/LocalMediaHandler.tsx

```diff
@@ -207,10 +207,18 @@ const LocalMediaHandler: FC<LocalMediaHandlerProps> = ({
 
 	useEffect(() => {
 		if (selectedDevicesId.audio === undefined && mediaAudioList[0]) {
-			setSelectedDevicesId({ audio: mediaAudioList[0].value, video: selectedDevicesId.video });
+			const defaultDevice = find(mediaAudioList, ['value', 'default']) ?? mediaAudioList[0];
+			setSelectedDevicesId({ audio: defaultDevice.value, video: selectedDevicesId.video });
```

Comment:
It's unclear why the video is also set when controlling the audio.
Also, adding selectedDevicesId to the dependency array triggers unnecessary execution when setSelectedDevicesId is called.
Consider using the function inside setSelectedDevicesId to retrieve the previous state and change only what's necessary.